bin/makeright

```diff
@@ -39,6 +39,9 @@
 #	X shared libraries are needed at run time.
 #
 # Hide X shared libraries from link libraries search path.
+
```

Comment:
Let's not separate the previous comment from the relevant line of code with this unrelated (to the comment) change.